POC/ColorPicker/ColorPicker/ColorPickingFunctionality/SystemEvents/MouseEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ColorPicker.ColorPickingFunctionality.SystemEvents
+{
+    class MouseEvent : SystemHook
+    {
+        private const int WH_MOUSE_LL = 14;
+        private const int WM_LBUTTONDOWN = 0x0201;
+
+        [StructLayout(LayoutKind.Sequential)]
+        private struct POINT
+        {
+            public int x;
+            public int y;
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        private struct MSLLHOOKSTRUCT
+        {
+            public POINT pt;
+            public uint mouseData;
+            public uint flags;
+            public uint time;
+            public IntPtr dwExtraInfo;
+        }
+
+        public delegate void EventCallBack(int x, int y);
```

Comment:
Callback should be one word. Also group public and private together (public first).